website/docs/components/pagination/partials/code/how-to-use.md

```diff
@@ -1,28 +1,28 @@
-## "Compact" vs "Numbered" pagination
+## Compact vs Numbered Pagination
 
 There are two different variants of the `Pagination` component (with different ways to invoke them) built to cover different use cases, contexts, and designs you may need them for.
 
-This differentiation is necessary to cover both use cases of a pagination for a list with a known number of elements (**"numbered"**) and one in which this information is not available or is [cursor-based](https://jsonapi.org/profiles/ethanresnick/cursor-pagination/) (**"compact"**).
+This differentiation is necessary to cover both use cases of pagination for a list with a known number of elements (i.e., "numbered") and one in which this information is not available or is [cursor-based](https://jsonapi.org/profiles/ethanresnick/cursor-pagination/) (i.e., "compact").
```

Comment:
@alex-ju I kept the quotes for these examples where it seemed it was introducing and explaining the concepts of numbered vs. compact vs talking about our components directly. I also added "ie" for clarity and removed bolding as I didn't think it was necessary.